src/code.rs

```diff
@@ -1,95 +1,82 @@
-    Blake2s256,
-    /// Strobe 256 (32-byte hash size)
-    #[cfg(feature = "strobe")]
-    #[mh(code = 0xa0, hasher = crate::Strobe256, digest = crate::StrobeDigest<crate::U32>)]
-    Strobe256,
-    /// Strobe 512 (64-byte hash size)
-    #[cfg(feature = "strobe")]
-    #[mh(code = 0xa1, hasher = crate::Strobe512, digest = crate::StrobeDigest<crate::U64>)]
-    Strobe512,
+#[derive(Clone, Debug, Eq, Multihash, PartialEq)]
+pub enum Multihash {
+    /// Multihash array for hash function.
+    #[mh(code = 0x00, hasher = crate::Identity256)]
+    Identity256(crate::IdentityDigest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x11, hasher = crate::Sha1)]
+    Sha1(crate::Sha1Digest<crate::U20>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x12, hasher = crate::Sha2_256)]
+    Sha2_256(crate::Sha2Digest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x13, hasher = crate::Sha2_512)]
+    Sha2_512(crate::Sha2Digest<crate::U64>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x17, hasher = crate::Sha3_224)]
+    Sha3_224(crate::Sha3Digest<crate::U28>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x16, hasher = crate::Sha3_256)]
+    Sha3_256(crate::Sha3Digest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x15, hasher = crate::Sha3_384)]
+    Sha3_384(crate::Sha3Digest<crate::U48>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x14, hasher = crate::Sha3_512)]
+    Sha3_512(crate::Sha3Digest<crate::U64>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x1a, hasher = crate::Keccak224)]
+    Keccak224(crate::KeccakDigest<crate::U28>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x1b, hasher = crate::Keccak256)]
+    Keccak256(crate::KeccakDigest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x1c, hasher = crate::Keccak384)]
+    Keccak384(crate::KeccakDigest<crate::U48>),
+    /// Multihash array for hash function.
+    #[mh(code = 0x1d, hasher = crate::Keccak512)]
+    Keccak512(crate::KeccakDigest<crate::U64>),
+    /// Multihash array for hash function.
+    #[mh(code = 0xb220, hasher = crate::Blake2b256)]
+    Blake2b256(crate::Blake2bDigest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0xb240, hasher = crate::Blake2b512)]
+    Blake2b512(crate::Blake2bDigest<crate::U64>),
+    /// Multihash array for hash function.
+    #[mh(code = 0xb250, hasher = crate::Blake2s128)]
+    Blake2s128(crate::Blake2sDigest<crate::U16>),
+    /// Multihash array for hash function.
+    #[mh(code = 0xb260, hasher = crate::Blake2s256)]
+    Blake2s256(crate::Blake2sDigest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0xa0, hasher = crate::Strobe256)]
+    Strobe256(crate::StrobeDigest<crate::U32>),
+    /// Multihash array for hash function.
+    #[mh(code = 0xa1, hasher = crate::Strobe512)]
```

Comment:
I guess you want to define the default codes as constants and use them here so:
`code = crate::STROBE_512`